src/index.css

```
@@ -86,3 +86,90 @@ blockquote {
 .components-popover__content {
 	width: 100%;
 }
+
+#assistant-markdown {
+	line-height: 1.6;
+	word-wrap: break-word;
+}
+
+#assistant-markdown blockquote {
+	font-weight: bold;
+	margin: 1rem 0;
+	padding: 0.5rem 1rem;
+}
+
+#assistant-markdown code {
+	border-radius: 3px;
+	margin: 1rem 0 !important;
+	white-space: pre-wrap;
+}
+
+#assistant-markdown h1 {
+	font-size: 1.5rem;
+	font-weight: bold;
+	margin-bottom: 0.5rem;
+	margin-top: 1.5rem;
+}
+
+#assistant-markdown h2 {
+	font-size: 1.25rem;
+	font-weight: bold;
+	margin-bottom: 0.5rem;
+	margin-top: 1.5rem;
+}
+
+#assistant-markdown h3 {
+	font-size: 1.125rem;
+	font-weight: bold;
+	margin-bottom: 0.5rem;
+	margin-top: 1.5rem;
+}
+
+#assistant-markdown h4 {
+	font-size: 1rem;
+	font-weight: bold;
+	margin-bottom: 0.5rem;
+	margin-top: 1.5rem;
+}
+
+#assistant-markdown h5 {
+	font-size: 0.875rem;
+	font-weight: bold;
+	margin-bottom: 0.5rem;
+	margin-top: 1.5rem;
+}
+
+#assistant-markdown h6 {
+	font-size: 0.85rem;
+	font-weight: bold;
+	margin-bottom: 0.5rem;
+	margin-top: 1.5rem;
+}
+
+#assistant-markdown hr {
+	border: none;
+	margin: 1rem 0;
+}
+
+#assistant-markdown ol {
+	list-style-type: decimal;
+	margin: 1rem 0;
+	padding-left: 1.5rem;
+}
+
+#assistant-markdown pre {
+	background-color: #272822;
```

Let's use the same background as the dymanic code block in the design.

```suggestion
	background-color: #1D2327;
```